src/Core/src/Platform/Android/Navigation/NavigationViewFragment.cs

```diff
@@ -58,6 +58,10 @@ public override AView OnCreateView(LayoutInflater inflater, ViewGroup? container
 					.CurrentPage
 					.ToPlatform(NavigationManager.MauiContext, RequireContext(), inflater, ChildFragmentManager);
 
+			// This shouldn't typically happen, but if a previous animation hasn't finished from a navigation that was interrupted
+			// the opacity of the view will be set to 0. This will reset it to 1.
+			NavigationManager.CurrentPage?.Handler?.UpdateValue(nameof(IView.Opacity));
```

Comment:
Could we detect when happens and only Invalidate the Opacity of the `CurrentPage `when is necessary?

Reply:
Yea, I thought about doing that but didn't want to try and get too clever. 

The opacity update is fairly small and only happens on a single navigation. Plus, I wanted the general opacity logic to trigger just to be safe.